tests/utils.py

```diff
@@ -143,7 +151,15 @@ def test_script_save(net, *inputs, eval_nets=True):
         result1 = net(*inputs)
         result2 = reloaded_net(*inputs)
 
```

Comment:
not 100% sure if it's relevant, perhaps we need to set the deterministic flag to ensure the reproducibility...
https://pytorch.org/docs/stable/notes/randomness.html
```suggestion
        monai.utils.set_determinism(seed=0)
        result1 = net(*inputs)
        result2 = reloaded_net(*inputs)
        monai.utils.set_determinism(seed=None)
```